build.gradle

```diff
@@ -10,11 +10,11 @@ ext {
 
     slf4jVersion        = '2.0.3'
     log4jVersion        = '2.19.0'
-    grpcVersion         = '1.48.1'
+    grpcVersion         = '1.51.0'
     protobufVersion     = '3.21.5'
     jacksonVersion      = '2.13.3'
     prometheusVersion   = '0.16.0'
-    cassandraDriver     = '4.14.1'
+    cassandraDriver     = '4.15.0'
```

Comment:
Please stick to `4.14.1` as official Datastax docs don't even have changelog for `4.15` version:
https://docs.datastax.com/en/developer/java-driver/4.14/changelog/
https://docs.datastax.com/en/developer/java-driver/